README.md

```diff
@@ -1 +1,55 @@
-# DATStar-Technology
\ No newline at end of file
+# Capstone Project
+
+> Desktop version + Mobile Version
+
+**This is a capstone project which is a website for DATStar Technology which is one of Vehicle GPS and Speedlimiter installation company.**
+
+## Built With
+
+- HTML
+- CSS
+- JavaScript
+
+## Materials required for validation
+- HTML, CSS and Javascript Linters
+- follow this link :
+ [link]( https://github.com/microverseinc/linters-config/tree/master/html-css)
+
+## Screenshot
+
+![screenshot_datstar](https://user-images.githubusercontent.com/38283436/147200947-44efbc44-e058-4206-afa1-ab2cb5e72a56.png)
+
+
+
```

Comment:
- [ ] BRILLIANT WORK SO FAR 💎. There is just one thing left to be done is creating a presentation video using [Loom.](https://www.loom.com/) I recommend creating a less than 5 mins short video about your awesome project, for more information please read [project requirements ](https://github.com/microverseinc/curriculum-html-css/blob/main/capstone/html_capstone.md)once again 👋🍀